Testnet incident release (#716)
Concordium/concordium.github.io: source/mainnet/net/resources/release-notes-mainnet.rst

   To subscribe to updates on the Mainnet status page click **Subscribe** to get all updates or click **Get updates** to choose to get all updates or only updates for specific products.

January 4, 2023

Concordium Node 5.0.7 for MacOS
-------------------------------

> A bug was fixed in the MacOS node that caused an issue with calculation of NRG.

Reviewer: ```suggestion
Fix a bug in the MacOS node that caused an issue with NRG calculation.
```

I think this is more in-line with how we write changelogs.
